src/pages/ChildrenDemo/ChildrenDemo.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { ThemeProvider } from '@material-ui/core';
+import Text from '../../components/Math/Math';
+import theme from '../../theme';
+import { Typography } from '@material-ui/core';
```

Comment:
lint your code: Absolute imports should be grouped together and relative imports together